tooling/template-base/types.ts

```diff
@@ -44,25 +44,27 @@ export type CurrentUser = {
 };
 
 export type PageContext = {
+  _pageId: string;
   Page: (pageProps: PageProps) => React.ReactElement;
   pageProps: PageProps;
-  pageExports: {
+  exports: {
```

Comment:
the updated `vite-plugin-ssr` deprecated several fields on the page context, including `pageExports`, `url`, `urlParsed.searchString`. These fields are changed to `exports`, `urlOriginal`, `urlParsed.searchOriginal` respectively. updated the references accordingly.